src/views/Compendium/Compendium.jsx

```diff
@@ -1,25 +1,33 @@
 import React from 'react';
 import { useState, useEffect } from 'react';
 import FilmList from '../../components/FilmList/FilmList';
-import { getFilms } from '../../services/data';
+import Controls from '../../components/Controls/Controls';
+import { filmDesc, getFilms } from '../../services/data';
 
 export default function Compendium() {
   const [films, setFilms] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [order, setOrder] = useState('asc');
 
   useEffect(() => {
     const fetchData = async () => {
-      const data = await getFilms();
+      let data;
+      if (order === 'desc') {
+        data = await filmDesc();
+      } else {
+        data = await getFilms();
+      }
```

Comment:
Ideally, the sort, if not supported by the api, is done via frontend state so we don't have to hit the api again. For the purpose of this project however, it's totally fine